docs/reference/logging/overview.md

```diff
@@ -107,6 +111,19 @@ the following named facilities:
 * `local6`
 * `local7`
 
+`syslog-tls-ca-cert` specifies the absolute path to the trust certificates
```

Comment:
Maybe form each of these into proper sentences:

  The `syslog-tls-ca-cert` options specifies ....